http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/LeoPubsubMessageSubscriber.scala

```diff
@@ -756,11 +766,13 @@ class LeoPubsubMessageSubscriber[F[_]: Timer: ContextShift: Parallel](
                   )
                 }
               }
-            monitorOp = gkeInterp.pollCluster(PollClusterParams(clusterId, msg.project, false, createClusterResult))
+            monitorOp = createClusterResultOpt.fold(F.unit)(createClusterResult =>
```

Comment:
If your `None` case is `F.unit` or `None` you can usually `traverse` instead of `fold`. :)

```
monitorOp = createClusterResult.traverse_ { createClusterResult =>
  gkeInterp.pollCluster(PollClusterParams(clusterId, msg.project, false, createClusterResult))
}
```